modules/base/src/algorithm/meshutils.cpp

```diff
@@ -321,33 +325,33 @@ std::shared_ptr<BasicMesh> colorsphere(const vec3& center, const float& radius,
         {std::atan(3.0f / 4.0f), 0.0f},
         {std::atan(2.0f / 5.0f), 0.0f},
         {std::atan(1.0f / 6.0f), 0.0f},
-        {M_PI_2, std::atan(1.0f / 6.0f)},
+        {glm::half_pi<float>(), std::atan(1.0f / 6.0f)},
         {std::atan(std::sqrt(26.0f)), std::atan(1.0f / 5.0f)},
         {std::atan(std::sqrt(17.0f) / 2.0f), std::atan(1.0f / 4.0f)},
         {std::atan(std::sqrt(10.0f) / 3.0f), std::atan(1.0f / 3.0f)},
         {std::atan(std::sqrt(5.0f) / 4.0f), std::atan(1.0f / 2.0f)},
-        {std::atan(std::sqrt(2.0f) / 5.0f), M_PI_4},
-        {std::atan(1.0f / 6.0f), M_PI_2},
-        {M_PI_2, std::atan(2.0f / 5.0f)},
+        {std::atan(std::sqrt(2.0f) / 5.0f), glm::quarter_pi<float>()},
```

Comment:
```suggestion
        {std::atan(glm::root_two<float>() / 5.0f), glm::quarter_pi<float>()},
```

glm has functions for sqrt of 2, 3, and 5.

Reply:
I think that just make it harder to read, for minute gains...